collectors/python.d.plugin/openldap/openldap.chart.py

```diff
@@ -153,6 +154,7 @@ def connect(self):
         try:
             self.conn = ldap.initialize('ldap://%s:%s' % (self.server, self.port))
             self.conn.simple_bind(self.username, self.password)
+            self.conn.set_option(ldap.OPT_NETWORK_TIMEOUT, self.timeout)
```

Comment:
please place it be before `simple_bind` call

Reply:
done